graphql-jpa-query-schema/src/main/java/com/introproventures/graphql/jpa/query/schema/impl/QraphQLJpaBaseDataFetcher.java

```diff
@@ -897,7 +918,29 @@ else if (value instanceof VariableReference) {
             }
         } else if (value instanceof ArrayValue) {
             Object convertedValue =  environment.getArgument(argument.getName());
-            if (convertedValue != null && !getJavaType(environment, argument).isEnum()) {
+            
```

Comment:
@igdianov what was the purpose of this change. I tried to run all the test with and without this change and they passed. Maybe we should add a test case for the issue solved by this change?

Reply:
@anotender The test case for this change already exists. I made the change to address the conversion of enum values arrays passed in the variable payload. i.e.

```
"genre": {
   "IN": ["NOVEL"]
 }
```

For some reason, Graphql Ast converter does not do that. I don't like the code, so I will try to refactor it to make it better.